subprojects/gradle/ci-logger/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
Create a single module for ci-logger. Before his classes were in two modules: `gradle:utils` and `gradle:logging`

Reply:
Is it a renamed `:gradle:logging` or a new one?

Reply:
As I wrote `ci-logger` is a new module created from `gradle:logging` and part of `gradle:utils`
`gradle:logging` was deleted